deploy/catalog_resources/courier/bundle_dir/0.4.0/broker_v1alpha1_activemqartemis_crd.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Assuming the content of the bundle directly is mostly duplicates, it would be good to use symlinks instead of maintaining copies of the files

Reply:
created symlinks for existing crd, csv and package file.